src/mod.ts

```diff
@@ -7,11 +7,11 @@ export const projectScripts = (args: string[]) => {
     "runtime": "deno",
     "hooks": {
       "get-manifest":
-        `deno run -q --config=deno.jsonc --allow-read --allow-net --allow-env https://deno.land/x/${HOOKS_TAG}/get_manifest.ts`,
+        `deno run -q --config=deno.jsonc --allow-read --allow-net --allow-env --allow-sys=onRelease https://deno.land/x/${HOOKS_TAG}/get_manifest.ts`,
```

Comment:
I think the `--allow-sys` settings here should match the ones from ROSI, but I'm not sure what they are

Reply:
Typo:

```suggestion
        `deno run -q --config=deno.jsonc --allow-read --allow-net --allow-env --allow-sys=osRelease https://deno.land/x/${HOOKS_TAG}/get_manifest.ts`,
```

Reply:
I think we should also add this to the start hook, and we probably need to add it as well to the files within `deno-slack-runtime`. I posted in the ROSI channel about doing so, let's see what shakes out there.